client/flutter/assets/content_bundles/symptom_checker.en.yaml

```
@@ -101,9 +104,38 @@ contents:
           body_html: |
             <p>No</p>
     -
-      id:  q4_age
+      id:  q4_ltc_hcf
       type: single_selection
-      display_condition: wasDisplayed('q3_covid19contact') && ((selectionCount('q2_symptoms') > 0) || q3_covid19contact['no_contact'])
+      display_condition: >-
+        wasDisplayed('q2_symptoms') &&
+        (selectionCount('q2_symptoms') > 0) &&
+        wasDisplayed('q3_covid19contact') &&
+        q3_covid19contact['no_contact']
+      question_html: |
+        <p><b>Are you a resident of or do you work in a healthcare facility or a longterm care facility?</b></p>
+      answers:
+        -
+          id: ltc_hcf
+          icon_name: snowflake
+          body_html: |
+            <p>Yes</p>
+        -
+          id: no_ltc_hcf
+          icon_name: snowflake
+          body_html: |
+            <p>No</p>
+    -
+      id:  q5_age
+      type: single_selection
+      display_condition: >-
+        wasDisplayed('q2_symptoms') && 
+        (
+          (q2_symptoms['fever'] && q2_symptoms['dry_cough']) ||
+          (wasDisplayed('q3_covid19contact') &&
+            ((selectionCount('q2_symptoms') > 0) ||
+              q3_covid19contact['no_contact'])
+            )
+        )
```

Can this now be updated to:

```suggestion
        (has('fever') && has('dry_cough')) ||
        (wasDisplayed('q3_covid19contact') &&
          ((selectionCount('q2_symptoms') > 0) ||
            q3_covid19contact['no_contact'])
          )
        )
```

---

no it would be true for emergencies.

---

Agree with our discussion on Slack that including `has('no_emergency')` will make this readable & accurate 😄